R/aggregate_results.R

```diff
@@ -207,14 +207,7 @@ aggregate_value_change_to_analysed_sectors_and_portfolio <- function(data,
       analysed_sectors_exposure = .data$asset_portfolio_value *
         sum(.data$plan_carsten, na.rm = TRUE),
       analysed_sectors_value_change = .data$analysed_sectors_exposure *
-        .data$VaR_analysed_sectors / 100,
-      portfolio_aum = .data$asset_portfolio_value,
```

Comment:
just to make sure before approving, are the columns that are no longer displayed to the users needed in earlier stages of the model? If not please make sure to delete their creation to declutter the code

Reply:
I'm not sure I follow. The variables are not used before in the model, only for aggregation. I am removing them here. Where else in the code should I declutter @MirijaHa ?

Reply:
I was being imprecise. What I am referring to is for example ```.data$asset_portfolio_value```. If that variable is not needed for the model itself its creation could be deleted earlier on.

Reply:
`.data$asset_portfolio_value` is needed in the calculation of the exposure of the sum of all analysed sectors above. That section will remain in the analysis. The column itself also provides valuable information, so I would prefer to leave the remaining columns as they are.